pact/provider/assertj/src/main/java/org/arquillian/algeron/pact/provider/assertj/PactProviderAssertions.java

```diff
@@ -0,0 +1,29 @@
+package org.arquillian.algeron.pact.provider.assertj;
+
+import org.arquillian.algeron.pact.provider.core.httptarget.Target;
+import org.assertj.core.api.Assertions;
+
+import java.net.URL;
+
+
+public class PactProviderAssertions extends Assertions {
+
+    Target target;
+
+    PactProviderAssertions(Target target) {
+        this.target = target;
+    }
+
+    public static PactProviderAssertions assertThat(Target target) {
+        return new PactProviderAssertions(target);
+    }
+
+    public void verifiesContract() {
```

Comment:
How about renaming this verification method, so instead of

```
assertThat(target).withUrl(.....).verifiesContract()
```

we could say

```
assertThat(target).withUrl(.....).satisfiesContracts()
```

I believe this wording fits better the overall assertThat - condition flow, rather than action which is how I understand `verifiesContract()`.